<launch>


    <arg name="UR_prefix" default="UR10_r"/>
    <arg name="tf_prefix" default="mur620a"/>
    <arg name="home_position" default="Home_custom"/>
    <arg name="move_group_name" default="UR_arm_r"/>

    <!-- enable UR -->
    <node name="UR_enable" pkg="ur_utilities" type="UR_enable.py" output="screen">
        <param name="ur_hardware_interface_topic" value="/$(arg tf_prefix)/$(arg UR_prefix)/ur_hardware_interface/set_mode" />
    </node>

    <node name="move_UR_to_home_position" pkg="ur_utilities" type="move_UR_to_home_position.py" output="screen">
        <param name="UR_prefix" value="$(arg UR_prefix)" />
        <param name="tf_prefix" value="$(arg tf_prefix)" />
        <param name="home_position" value="$(arg home_position)" />
        <param name="move_group_name" value="$(arg move_group_name)" />
    </node>


</launch>
